docs/src/main/paradox/user/lang/kotlin/gettingstarted.md

```diff
@@ -0,0 +1,288 @@
+# Getting started with stateful services in Kotlin
+
+## Prerequisites
+
+Kotlin version
+: Cloudstate Kotlin support requires at least Java 8, though we recommend using Java 11, which has better support for running in containers. While it is possible to build a GraalVM native image for Cloudstate Kotlin user functions, at this stage Cloudstate offers no specific assistance or configuration to aid in doing that.
+
+Build tool
+: Cloudstate does not require any particular build tool, you can select your own. Or use [Cloudstate CLI](https://github.com/sleipnir/cloudstate-cli)
```

Comment:
missing . at the end of sentence:

Or use **the** [Cloudstate CLI](https://github.com/sleipnir/cloudstate-cli)**.**